src/main/java/de/retest/web/AttributesConfig.java

```diff
@@ -17,6 +19,13 @@
 	public static final String WIDTH = "width";
 	public static final String HEIGHT = "height";
 
+	public static final String TAG_NAME = "tagName";
```

Comment:
Isn't tag name = type?

Reply:
We call it type but actaully use tagName to extract it from the data.